Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/BehaviorData.xml

```diff
@@ -7174,4 +7174,22 @@
         <Period value="0.75"/>
         <Requirements value="AP_NotHaveK5CreepBonuses"/>
     </CBehaviorCreepSource>
+    <CBehaviorBuff id="AP_MercCompoundInfantry">
+        <InfoFlags index="Hidden" value="1"/>
+    </CBehaviorBuff>
+    <CBehaviorBuff id="AP_MercCompoundVehicles">
+        <InfoFlags index="Hidden" value="1"/>
+    </CBehaviorBuff>
+    <CBehaviorBuff id="AP_MercCompoundStarships">
+        <InfoFlags index="Hidden" value="1"/>
+    </CBehaviorBuff>
+    <CBehaviorBuff id="AP_MercCompoundTabNotSelected">
+        <InfoFlags index="Hidden" value="1"/>
+        <EditorCategories value="AbilityorEffectType:Units"/>
+        <RemoveValidatorArray value="AP_NotMercCompoundInfantryTabSelected"/>
+        <RemoveValidatorArray value="AP_NotMercCompoundVehiclesTabSelected"/>
+        <RemoveValidatorArray value="AP_NotMercCompoundStarshipsTabSelected"/>
+        <Period value="1"/>
+        <PeriodicEffect value="AP_MercCompoundTabSearchSwitch"/>
+    </CBehaviorBuff>

```

Comment:
Considering there's nothing _on_ the misc tab, maybe it's better to default to the infantry tab like Waikin had?

Reply:
The first tab that gets available is opened. So if you have only starships, the starship tab will open on mission start. This basically waits for anything getting available